<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>6</y>
      <width>271</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Путь к файлу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file_inpt"/>
       </item>
       <item>
        <widget class="QPushButton" name="choose_btn">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="file_btn">
           <property name="text">
            <string>Импортировать точки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="export_btn">
           <property name="text">
            <string>Экспортировать точки</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Перенос</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>dx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dx_inpt"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>dy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dy_inpt"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>dz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dz_inpt"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="translate_btn">
         <property name="text">
          <string>Перенести модель</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Поворот на угол</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>dx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="x_angle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>dy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="y_angle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>dz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="z_angle"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="rotate_btn">
         <property name="text">
          <string>Повернуть модель</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Масштабирование</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>sx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="x_scale"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>sy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="y_scale"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>sz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="z_scale"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="scale_btn">
         <property name="text">
          <string>Масштабировать модель</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="draw_btn">
         <property name="text">
          <string>Нарисовать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graph">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>571</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
